includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -2184,6 +2185,9 @@ public function update_order_status() {
 			}
 
 			$payment_method_id = isset( $_POST['payment_method_id'] ) ? wc_clean( wp_unslash( $_POST['payment_method_id'] ) ) : '';
+			if ( 'null' === $payment_method_id ) {
+				$payment_method_id = '';
+			}
```

Comment:
This was a weird little bug I ran into and I'm not exactly sure how it happens. But for whatever reason the value of `$_POST['payment_method_id']` was set to `"null"` (i.e. a string) when it was supposed to be `null`.

Honestly not sure how to reproduce, but adding the extra if-statement here fixed the issue.